proto/cosmwasm/wasm/v1/proposal.proto

```diff
@@ -20,11 +20,8 @@ message StoreCodeProposal {
   string run_as = 3;
   // WASMByteCode can be raw or gzip compressed
   bytes wasm_byte_code = 4 [ (gogoproto.customname) = "WASMByteCode" ];
-  // Source is a valid absolute HTTPS URI to the contract's source code,
-  // optional
-  string source = 5;
-  // Builder is a valid docker image name with tag, optional
-  string builder = 6;
+  // 5 and 6 intentionally left empty (used in v1beta1)
```

Comment:
Maybe this leads to better doc generation? See https://developers.google.com/protocol-buffers/docs/proto#reserved

```suggestion
  // Used in v1beta1
  reserved 5, 6;
```

Reply:
I vaguely remembered there was some way to mark it as reserved in protobuf.
Nice find.

Reply:
I just did that fix-up